dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/report/MetadataReportInstance.java

```diff
@@ -110,6 +110,8 @@ private void init(MetadataReportConfig config, MetadataReportFactory metadataRep
         // exist."));
         MetadataReport metadataReport = metadataReportFactory.getMetadataReport(url);
         if (metadataReport != null) {
+            System.out.println("MetadataReportFactory:"+ metadataReportFactory.getClass()+",MetadataReport:"+metadataReport+" url:"+url+" ,relatedRegistryId:"+metadataReport.relatedRegistryIdSuffix(url));
```

Comment:
Sout is not allow

Reply:
Just for temporary test,I will remove it later